db.go

```diff
@@ -528,6 +530,47 @@ func (db *DB) Close() error {
 	return merr.Err()
 }
 
+// DisableCompactions disables compactions.
+func (db *DB) DisableCompactions() error {
```

Comment:
Hmm, so I remove the error for now? If we add contexts and we timeout, we would need to return an error then right?